### ISSUE TITLE
[SUGGESTION]: Increase Battle Cafe max spins for resistant/shiny mons

### ISSUE THREAD

**Issue by Jelith** [pokeclicker/pokeclicker#4362]
### Current Behavior

Once all 9 Alcremie in a berry category are Caught, daily spins increases by 1.

Once all 9 Alcremie in a berry category become Shiny, daily spins increases by 1.

Once all 9 Alcremie in a berry category become Resistant, daily spins increases by 1.

Given this, it takes, playing optimally, approximately 30 days to make all 63 Alcremie resistant.

### Improved Behavior

Once all 9 Alcremie in a berry category are Caught, daily spins increases by 1.

Once all 9 Alcremie in a berry category become Shiny, daily spins increases by 2.

Once all 9 Alcremie in a berry category become Resistant, daily spins increases by 3.

With this change, it will take approximately 18 days to make all 63 Alcremie resistant.

### Reason for change

Players are already forced to farm berries for months in order to meet the requirements to resist all 63 Alcremies. This would save a couple of weeks on top of that, while also making the player feel much more rewarded for hitting Shiny and Resistant milestones.

### Detailed Description

Self-explanatory.

**Comment by Farboo171** [comment]
Is this not #3760 somehow?

**Comment by Jelith** [comment]
> Is this not #3760 somehow?

No, it appears that issue was adopted to include the current behavior of increasing max by 1 each for each shiny.
